pkg/karmadactl/cmdinit/cmdinit.go

```diff
@@ -102,6 +103,7 @@ func NewCmdInit(parentCommand string) *cobra.Command {
 	// cert
 	flags.StringVar(&opts.ExternalIP, "cert-external-ip", "", "the external IP of Karmada certificate (e.g 192.168.1.2,172.16.1.2)")
 	flags.StringVar(&opts.ExternalDNS, "cert-external-dns", "", "the external DNS of Karmada certificate (e.g localhost,localhost.com)")
+	flags.DurationVar(&opts.ValidityPeriod, "cert-validity-period", cert.Duration365d, "the validity period of Karmada certificate (e.g 8760h0m0s: default 365d)")
```

Comment:
```suggestion
	flags.DurationVar(&opts.ValidityPeriod, "cert-validity-period", cert.Duration365d, "The validity period of Karmada certificate (e.g 8760h0m0s: default 365d)")
```
Does `Karmada cerificate` means all signed certificates, including both `client` and `server`?

By the way, we don't need to specify the default value again after we set the default value. The framework would set the default value.

Reply:
> e.g 8760h0m0s
Can people set the flag by `year` /`month`/`day`?

Reply:
I suggest renaming the `cert.Duration365d` to `cert.DefaultCertificateValidity`.

Reply:
> Does Karmada cerificate means all signed certificates, including both client and server?

Yes, include certs of all components.

> Can people set the flag by `year` /`month`/`day`?

It seems not working, it is only recognizing the default format(hour/minute/second) of time.Duration. It should be more flexible about fine grained time.

> By the way, we don't need to specify the default value again after we set the default value. The framework would set the default value.

Yes, but `(e.g 8760h0m0s: default 365d)` is only for commenting `8760h0m0s == 365d`.

Reply:
OK. get it. Please echo the output of `karmadactl init --help`, especially for this flag part.
Maybe we can comment it like `e.g 8760h0m0s, that is 365d`.

Reply:
```
Options:
    --cert-external-dns='':
        the external DNS of Karmada certificate (e.g localhost,localhost.com)

    --cert-external-ip='':
        the external IP of Karmada certificate (e.g 192.168.1.2,172.16.1.2)

    --cert-validity-period=8760h0m0s:
        the validity period of Karmada certificate (e.g 8760h0m0s, that is 365 days)
```